x-pack/plugins/apm/ftr_e2e/cypress/e2e/power_user/service_overview/mobile_overview_with_most_used_charts.cy.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import url from 'url';
+import moment from 'moment/moment';
+import { synthtrace } from '../../../../synthtrace';
+import { generateMobileData } from './generate_mobile.data';
+
+const start = Date.now() - 1000;
+const end = Date.now();
+
+const rangeFrom = new Date(start).toISOString();
+const rangeTo = new Date(end).toISOString();
+
+const apiRequestsToIntercept = [
+  {
+    endpoint: '/internal/apm/mobile-services/synth-android/most_used_charts?*',
+    aliasName: 'mostUsedChartRequest',
+  },
+];
+
+const aliasNames = apiRequestsToIntercept.map(
+  ({ aliasName }) => `@${aliasName}`
+);
+
+const apmMobileServiceOverview = url.format({
+  pathname: 'app/apm/mobile-services/synth-android',
+  query: {
+    rangeFrom,
+    rangeTo,
+  },
+});
+describe('Mobile Service overview page', () => {
+  before(() => {
+    synthtrace.index(
+      generateMobileData({
+        from: new Date(start).getTime(),
+        to: new Date(end).getTime(),
+      })
+    );
+  });
+
+  after(() => {
+    synthtrace.clean();
+  });
+
+  describe('Mobile service overview with charts', () => {
+    beforeEach(() => {
+      cy.loginAsEditorUser();
+      cy.visitKibana(apmMobileServiceOverview);
+      apiRequestsToIntercept.map(({ endpoint, aliasName }) => {
+        cy.intercept('GET', endpoint).as(aliasName);
+      });
+    });
+
+    describe('accessing android service page', () => {
+      it('shows the mobile service overview page with default params and transactionType as Mobile', () => {
+        cy.location().should((loc) => {
+          expect(loc.pathname).to.eq(
+            '/app/apm/mobile-services/synth-android/overview'
+          );
+          expect(loc.search).to.eq(
+            `?comparisonEnabled=true&environment=ENVIRONMENT_ALL&kuery=&latencyAggregationType=avg&rangeFrom=${rangeFrom}&rangeTo=${rangeTo}&serviceGroup=&offset=1d&transactionType=mobile`
+          );
+        });
+      });
+
+      it('shows the most used charts', () => {
+        cy.wait(aliasNames);
+
+        cy.getByTestSubj('mostUsedChart-device').should('exist');
```

Comment:
I think even just calling `cy.getByTestSubj('mostUsedChart-device')` fails / timeouts if the element is not there

Reply:
Agreed, but i preferred the explicit `.should('exist')` for better readability of code.